app/routes/app/AppRoute.js

```diff
@@ -70,59 +71,61 @@ type AppProps = any;
 class App extends PureComponent<AppProps> {
   render() {
     return (
-      <div
-        className={cx(styles.appRoute, {
-          [styles.searchOpen]: this.props.searchOpen
-        })}
-      >
-        <Helmet defaultTitle="Abakus.no" titleTemplate="%s | Abakus.no">
-          <meta property="og:image" content={coverPhoto} />
-          <meta
-            property="og:description"
-            content="Abakus er linjeforeningen for studentene ved Datateknologi og Kommunikasjonsteknologi på NTNU, og drives av studenter ved disse studiene."
-          />
-        </Helmet>
+      <SpecialDay>
```

Comment:
@odinuge @orhanhenrik Is this a "legal" way to implement something like this in React with SSR? I'm using browser Local Storage and it's working with SSR, since I only use it in `componentDidMount`. I'm just unsure if this implementation is the proper way.